### ISSUE TITLE
Form Fields should only be placeable as an descendant of the Form plugin

### ISSUE THREAD

**Issue by stefanfoulis** [aldryn/aldryn-forms#1]

**Comment by DmitriySelischev** [comment]
What if I have to implement special markup(such as multicolumn) under my form for fields? What should I do? I really have such case. Guys, you are the main developers, but I think that "enhancement" should be called limitation! Please, don't do this.

**Comment by DmitriySelischev** [comment]
Is that means that we can place field under form but in several levels of containers?

**Comment by czpython** [comment]
Yes. This doesn't mean that fields can't have multicolumn or other plugins as parents, this means that fields should always be **descendants** of the form plugin, doesn't matter where in the tree.

**Comment by DmitriySelischev** [comment]
Thanks for your answer. Good to know

**Comment by treavis** [comment]
Could this be reverted to [467adbd](https://github.com/aldryn/aldryn-forms/commit/467adbd5bf7bdf48a784aff768019fa3008d7fee) commit, because it causes more problems than it solves? If you, by accident, place form field outside the form plugin (which is now possible), it causes database error.

**Comment by czpython** [comment]
Reverting to that commit would be a major backwards incompatible change and not really aligned with the design of this app. There are more occurrences of users adding non-field plugins under the form than users adding the form fields accidentally outside the form.

**Comment by treavis** [comment]
Ok. Good to know. I'll keep using my fork. Just asked because this actually happened, and it could only be fixed by manually removing entries from database.

**Comment by czpython** [comment]
On cms 3.5 we're introducing the `?structure` GET parameter which is for structure-only view, hopefully this makes it easier to correct these issues without having to do so manually on the db.

**Comment by zbohm** [comment]
I am afraid that idea of placing field only under some form plugin is not good. There may be cases where the user wants to place the field outside the form. And input outside form is perfectly html valid code.

https://validator.w3.org/:
```html
<!doctype html>
<html lang="en">
<head>
    <title>Title</title>
</head>
<body>
   <input>
</body>
</html>
```
